src/Core/src/PropertyMapperExtensions.cs

```diff
@@ -26,6 +26,31 @@ void newMethod(TViewHandler handler, TVirtualView view)
 			propertyMapper.Add(key, newMethod);
 		}
 
+		/// <summary>
+		/// Modify a property mapping in place but call the previous mapping if the types do not match.
+		/// </summary>
+		/// <typeparam name="TVirtualView">The cross-platform type.</typeparam>
+		/// <typeparam name="TViewHandler">The handler type.</typeparam>
+		/// <param name="propertyMapper">The property mapper in which to change the mapping.</param>
+		/// <param name="key">The name of the property.</param>
+		/// <param name="method">The modified method to call when the property is updated.</param>
+		internal static void ModifyMappingWhen<TVirtualView, TViewHandler>(this IPropertyMapper<IElement, IElementHandler> propertyMapper,
```

Comment:
These methods can be public I suppose

Reply:
These new methods are the real work. They basically only run if the types match, otherwise they fall back to the "base" implementation.